deploy.ps1

```diff
@@ -35,18 +35,32 @@ $subModuleContent = $subModuleContent.replace('${version}', $config.version)
 $subModuleContent = $subModuleContent.replace('${game_version}', $config.game_version)
 $subModuleContent | Out-File -Encoding utf8 -FilePath $DeployDir\SubModule.xml
 
-
 # copy mod dll
-Copy-item -Force *.dll -Destination $DeployBinDir
+Copy-Item -Force *.dll -Destination $DeployBinDir
+
+# copy game dlls for deployment
+Copy-Item ${BaseDir}mb2 -Force -Filter "TaleWorlds*.dll" -Destination $DeployDir -Recurse
+Copy-Item ${BaseDir}mb2 -Force -Filter "SandBox*.dll" -Destination $DeployDir -Recurse
```

Comment:
This is not needed as the dlls are copied to the build directory automatically
```suggestion
```

Reply:
Now I'm doing this in a separate ps1 script, as Dockerfile cannot access files inside the mb2 symlink